docs/setup-cluster/workspaces.rst

```diff
@@ -35,31 +35,30 @@ individuals or teams using Determined. This is recommended for larger teams.
 WebUI
 =====
 
-After logging in, the default landing page for the WebUI is **Uncategorized Experiments List**. This
-is where experiments that do not have a configuration file ``workspace`` and ``project`` are put,
-including experiments created before the workspaces feature was released.
+After logging in, the dashboard displays. This is the default landing page for the WebUI. The
```

Comment:
I don't think this sentence is correct? Maybe you want "After logging in you will be see the dashboard"? The comma here is incorrect and the the object/subject arrangement doesn't agree with the rest of the paragraph.